lib/service/storage/elasticsearch.js

```diff
@@ -210,6 +210,54 @@ class ElasticSearch extends Service {
       .catch(error => this._esWrapper.reject(error));
   }
 
+  /**
+   * Returns detailed multi-level storage stats data
+   *
+   * @returns {Promise.<Object>}
+   */
+  async stats () {
+    const esRequest = {
+      metric: ['docs', 'store'],
+    };
+
+    const { body } = await this._client.indices.stats(esRequest);
+
+    const response = {
+      indexes: {},
+      size_in_bytes: 0,
```

Comment:
Why using `snake_case` here?

Reply:
Got stuck in the head with the idea of having the most 'es' like response, but indeed we don't need to :face_with_thermometer: